src/components/annotate/sidePanel/Annotation.tsx

```diff
@@ -0,0 +1,200 @@
+    });
+
+    return (
+        <Row className="mb-3">
+            <Col>
+                <div className={`${styles.sidePanelAnnotation} px-3 py-2`} ref={annotationRef}>
+                    {/* Creator and date */}
+                    <Row>
+                        <Col>
+                            <p className={classCreator}> {annotation.creator} </p>
+                        </Col>
+                        <Col className="col-md-auto">
+                            <p className="c-primary"> {Moment(annotation.created).format('MMMM DD - YYYY')} </p>
+                        </Col>
+                    </Row>
+                    {/* Annotation content */}
+                    {!annotateTarget.property &&
+                        <Row className="my-2">
+                            <Col className="col-md-auto pe-0">
+                                <div className={`${styles.sidePanelTopStripe} h-100`} />
+                            </Col>
+                            <Col>
+                                <p className="fst-italic"> {annotation.target.indvProp} </p>
+                            </Col>
+                        </Row>
+                    }
+                    <Row className="mt-1">
+                        <Col>
+                            <p>
+                                <span className="c-primary">
+                                    {`${annotationMotivations[annotation.motivation as keyof typeof annotationMotivations].displayName}: `}
+                                </span>
+                                {annotationValue}
+                            </p>
+                        </Col>
+                    </Row>
+                    {/* Comments and actions */}
+                    <Row className="mt-2">
+                        <Col className="col-md-auto">
+                            <p className="c-grey"> 0 comments </p>
+                        </Col>
+                        <Col className="col-md-auto ps-1">
+                            <FontAwesomeIcon icon={faThumbsUp}
+                                className="c-pointer c-primary"
+                            />
+                        </Col>
+                        <Col className="align-items-center ps-0">
+                            <FontAwesomeIcon icon={faThumbsDown}
+                                className="c-pointer c-primary"
+                            />
+                        </Col>
+
+                        {KeycloakService.GetSubject() === annotation.creator &&
+                            <Col className="col-md-auto">
+                                <Row>
+                                    <Col className="pe-0">
+                                        <FontAwesomeIcon icon={faPen}
+                                            className="c-pointer c-primary"
+                                            onClick={() => dispatch(setEditAnnotation(annotation))}
```

Comment:
The user should only be able to update/delete their own annotation. Do we check this in the FE?

Reply:
Yes. The edit and delete icons are only generated when the user is logged in and if the creator id of the annotation is the same as the user's keycloak id.

![image](https://github.com/DiSSCo/disscover/assets/102812384/aa4b16bd-01f0-444d-9bb6-d2ae2b7c3709)

Reply:
Check, thanks for explaining